pkg/monitors/uptimerobot/uptime-monitor_test.go

```diff
@@ -387,39 +394,42 @@ func TestAddMonitorWithIncorrectValues(t *testing.T) {
 	}
 }
 
-func TestAddMonitorWithAlertContacts(t *testing.T) {
-	config := config.GetControllerConfigTest()
+// disabling this test since Free Tier UptimeRobot doesnt allow to add Alert Contacts
```

Comment:
Maybe we should have it as a toggle instead, depending on what version of UptimeRobot we use?

Reply:
> Maybe we should have it as a toggle instead, depending on what version of UptimeRobot we use?
Actually, that cannot be verified because token is checked on runtime by the UptimeRobot website, about the access/tier the token has. so we will have to comment this test out for now